packages/web-twig/src/Resources/components/Header/HeaderDialogCloseButton.twig

```diff
@@ -0,0 +1,24 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _class = props.class | default(null) -%}
+{%- set _dialogId = props.dialogId -%}
+{%- set _label = props.label | default('Close') -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'HeaderDialogCloseButton' -%}
+
+{# Miscellaneous #}
+{%- set _classNames = [ _rootClassName, _class ] -%}
+
+<button
+    {{ mainProps(props) }}
+    {{ classProp(_classNames) }}
+    type="button"
+    data-dismiss="offcanvas"
```

Comment:
I think we have here the same problem as in ModalHeader.

Reply:
That is…?

Reply:
Using the `enableDismiss` that you are setting here https://github.com/lmc-eu/spirit-design-system/pull/712/files#diff-bf7a7f28e378fe3c98293de68e58dab13572b3f489ad36eb70d7f69b69bf0520R5.

Reply:
Maybe, because this is a new implementation, it can be `false` by default.

Reply:
Agreed we will now add the `enableDismiss` property here, but with the default value of `true` so it's not a BC for `ModalHeader` where the same property already exists.